pkg/client/unversioned/request.go

```diff
@@ -98,6 +98,13 @@ type Request struct {
 
 	apiVersion string
 
+	// group/version
+	group      string
+	version    string
```

Comment:
`apiVersion` is currently not used in Request. Both `baseURL` and `apiVersion` are derived from RESTClient when it creates a new Request, and neither `apiVersion` nor `group`/`version` affects `baseURL` in Request. This is the first step before we remove references to "version" from the RESTClient. 

`UnversionedPath` strips out the version word (such as `v1`) in `baseURL` and save that to Request `path` (doesn't change `baseURL`), and `path` and `baseURL` are used for generating a final url in URL().